src/styles/auto-prefix.js

```diff
@@ -20,18 +20,29 @@ export default {
     let prefixer = prefixers[userAgent];
     // None found, create a new instance
     if (!prefixer) {
-      prefixer = new InlineStylePrefixer(userAgent);
+      prefixer = new InlineStylePrefixer({userAgent: userAgent});
```

Comment:
@oliviertassinari Personally, I've come to like it since its shorter and "looks" more like ES6. That being said, I'm biased because I use Airbnb's eslint configuration in my projects :smile:.

I just added a reminder comment about it in #2903 so we can discuss and decide there.